transformerlab/routers/tasks.py

```diff
@@ -0,0 +1,103 @@
+import json
+from fastapi import APIRouter
+
+import transformerlab.db as db
+
+router = APIRouter(prefix="/tasks", tags=["tasks"])
+
+
+@router.get("/list")
+async def tasks_get_all():
+    tasks = await db.tasks_get_all()
+    return tasks
+
+@router.get("/list_by_type")
+async def tasks_get_by_type(type: str):
+    tasks = await db.tasks_get_by_type(type)
+    return tasks
+
+@router.get("/delete_all")
+async def tasks_delete_all():
+    await db.tasks_delete_all()
+    return {"message":"OK"}
+
+@router.get("/convert_training_template_to_task")
```

Comment:
You have to document these functions and routes so that we know why we made them later on